tests/integration/pipeline.ts

```diff
@@ -601,6 +601,17 @@ describe('Pipeline Editor tests', () => {
     cy.findByRole('button', { name: /cancel/i }).click();
   });
 
+  it('airflow pipeline should display error dialog if username and password are not filled in', () => {
+    cy.createPipeline({ type: 'airflow' });
+    cy.savePipeline();
```

Comment:
I guess we don't need to create a pipeline if we are just testing saving a new runtime